example_template.md

```diff
@@ -1,38 +1,41 @@
-**Table of Contents**
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-- [Run the EXAMPLE NAME](#run-the-example-name)
-  - [🗝️ Pre requirements](#Pre-requirements)
-  - [🔮 Overview of the files](#Overview-of-the-files)
-  - [🏃 Run the Flows](#run-the-flows)
-  - [🌀 Flow Diagram](#flow-diagram)
-  - [🌟 Results](#results)
-  - [🧞 OPTIONAL: EXTRA KNOWLEDGE FROM THIS EXAMPLE](#optional)
-  - [💫 Deploy with Docker](#Deploy-with-docker)
-  - [Next steps](#next-steps)
-  - [Community](#community)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
 # Run the EXAMPLE NAME
+| About this example: |  |
+| ------------- | ------------- |
+| Learnings | *Describe what the user will learn after running this example* |
+| Used for indexing | *What is the datatype of the indexing input* |
+| Used for querying | *What is the data type of the query input* |
+| Dataset used | *Link to the datasets* |
+| Model used | *Link to the model* |
+
+<p></p>
+This example shows you how to...
+
+*Add a short description of*
+1. *What is this example?*
+2. *What is the expected outcome of this example?*
+3. *What the user will learn in this example?*
+
+*You can also include a gif with a full demo of the example*
+_____
+
+## 🐍 Build the app with Python
+
+These instructions explain how to build the example yourself and deploy it with Python. If you want to skip the building steps and just run the example with Docker, check [the Docker deployment instructions at the end of this README](#deploy-with-docker)  
 
-Write a short description of
-1. What is this?
-2. What is the expected outcome?
-3. What the user will learn?
 
-If you want to run the example with Docker, check the instruction at the end of this README.  
+### 🗝️ Pre-requirements
 
-## 🗝️ Pre requirements
+*Here outline in bullet points anything the user is expected to have before diving in.* 
 
-Outline in bullet points anything the user is expected to have before diving in. For example:
+For example:
 
 1. You have a working Python 3.8 environment. 
-2. We recommend creating a [new python virtual envoriment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency clashing.   
-3. You have at least 8GB of free space on your hard drive. 
+2. We recommend creating a [new Python virtual environment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency conflicts.   
+3. You have at least 2GB of free space on your hard drive. 
```

Comment:
@FionnD Why is it 2GB? Does it depend on the size of the examples repo?

Reply:
No idea. I didn't make any changes to that bit

Reply:
It was 2GB for the example I was working with, but it would be dependent on each example